src/main/webapp/view/index.html

```diff
@@ -33,10 +33,71 @@ <h2 id="header-text">ZoomTube</h2>
+              />
+            </svg>
+          </button>
+          <button class="icon-button" type="button" onclick="iconOnClick('TOO_FAST')">
+            <svg
+              width="2.5em"
+              height="2.5em"
+              viewBox="0 0 16 16"
+              class="bi bi-skip-forward"
+              fill="currentColor"
+              xmlns="http://www.w3.org/2000/svg"
+              focusable="false"
+            >
+              <path
+                fill-rule="evenodd"
+                d="M15.5 3.5a.5.5 0 0 1 .5.5v8a.5.5 0 0 1-1 0V8.752l-6.267 3.636c-.52.302-1.233-.043-1.233-.696v-2.94l-6.267 3.636C.713 12.69 0 12.345 0 11.692V4.308c0-.653.713-.998 1.233-.696L7.5 7.248v-2.94c0-.653.713-.998 1.233-.696L15 7.248V4a.5.5 0 0 1 .5-.5zM1 4.633v6.734L6.804 8 1 4.633zm7.5 0v6.734L14.304 8 8.5 4.633z"
+              />
+            </svg>
+          </button>
+          <button class="icon-button" type="button" onclick="iconOnClick('TOO_SLOW')">
+            <svg
+              width="2.5em"
+              height="2.5em"
+              viewBox="0 0 16 16"
+              class="bi bi-skip-backward"
+              fill="currentColor"
+              xmlns="http://www.w3.org/2000/svg"
+              focusable="false"
+            >
+              <path
+                fill-rule="evenodd"
+                d="M.5 3.5A.5.5 0 0 1 1 4v3.248l6.267-3.636c.52-.302 1.233.043 1.233.696v2.94l6.267-3.636c.52-.302 1.233.043 1.233.696v7.384c0 .653-.713.998-1.233.696L8.5 8.752v2.94c0 .653-.713.998-1.233.696L1 8.752V12a.5.5 0 0 1-1 0V4a.5.5 0 0 1 .5-.5zm7 1.133L1.696 8 7.5 11.367V4.633zm7.5 0L9.196 8 15 11.367V4.633z"
```

Comment:
Idk what these things mean. Can you create consts in HTML to store these values? If not, then I would add a comment.

Reply:
+1 Please add a comment describing what this is.

Reply:
Added comment!

Reply:
These are all different paths in your comment can you explain which icon it is? Example: <!-- Coordinate path of thumbs up icon svg . -->

Reply:
Added